packages/shared/src/utils/constants/default-options.ts

```diff
@@ -15,3 +16,8 @@ export const getIDVNotApplicableOption = (is_for_real_account_signup_modal?: boo
  * Returns default value for the text to render when there are no matching results.
  */
 export const getSearchNotFoundOption = () => localize('No results found');
+
+/**
+ * Returns List of unsupported languages based on the environment.
+ */
+export const BLOCK_LANGUAGES = isProduction() ? ['ID'] : [];
```

Comment:
Can you remind about the logic behind this?
Also rename to "BLOCKED_LANGUAGES" or "UNSUPPORTED"

Reply:
`getAllowedLanguages` implementation in Deriv-app (`packages/translations/src/i18next/i18next.ts`) has a condition to filter languages based on the environment.  The same function will now come from @deriv-com/translations lib. So only the input to filterout languages can be handled by the consumer. I will rename the variable